testing/integration/proxy_url_test.go

```diff
@@ -392,19 +392,29 @@ func TestProxyURL_RemoveProxyFromThePolicy(t *testing.T) {
 	p.assertConnectedFleet(t)
 }
 
+// assertConnectedFleet is deprecated, use AssertConnectedToFleet instead
+// Deprecated.
 func (p *ProxyURL) assertConnectedFleet(t *testing.T) {
+	AssertConnectedToFleet(t, p.fixture)
+}
+
```

Comment:
Why not just remove this function and replace all its calls with `AssertConnectedToFleet`? From what I can tell, this method is only being called from within `proxy_url_test.go`. Is it to keep the size of this PR small? In that case I'd do this cleanup in an immediate follow up PR so we don't forget about it.

Reply:
it was temporary while I was checking exactly what the test was intended to do and if my fix worked. I'm fixing it all now